web-common/src/features/dashboards/state-managers/actions/dimension-filters.ts

```diff
@@ -38,6 +38,30 @@ export function toggleDimensionValueSelection(
   }
 }
 
+export function toggleDimensionNameSelection(
```

Comment:
I am not sure if this is a toggle anymore. The above one either adds or removes existing one. This just adds an entry. Not a 100% sure what is a good name though.

Reply:
Made it a toggle